infra/base-images/base-builder/Dockerfile

```diff
@@ -149,7 +149,8 @@ RUN mkdir honggfuzz && \
     rm -rf $SRC/oss-fuzz.tar.gz
 
 COPY compile compile_afl compile_dataflow compile_libfuzzer compile_honggfuzz \
-    compile_go_fuzzer precompile_honggfuzz srcmap write_labels.py /usr/local/bin/
+    compile_go_fuzzer cargo precompile_honggfuzz srcmap \
```

Comment:
nit: indent right to compile in previous line.
nit: move cargo to start, alpha order

Reply:
ok